datacenter/dtr/2.1/guides/release-notes.md

```diff
@@ -13,6 +13,23 @@ known issues for each DTR version.
 You can then use [the upgrade instructions](install/upgrade.md),
 to upgrade your installation to the latest release.
 
+## DTR 2.1.2
+
+(8 Dec 2016)
+
+**Features**
+
+* The web UI now alerts when no backups have been made in a week
+* Google Cloud Storage driver now throttles data if there's heavy load
+
+
+**Bug fixes**
+
+* Restore operation now prints logs
+* Upgraded alpine images used by the DTR services to fix security vulnerabilities
```

Comment:
Only the expat2 library is changed.  The problem is there are false positives right now with two other packages which can be confusing to users who think we have vulnerabilities if you don't call out which CVE was fixed.